substratevm/src/com.oracle.svm.core/src/com/oracle/svm/core/code/CompilationResultFrameTree.java

```diff
@@ -278,6 +284,48 @@ public String toString() {
             return String.format("-%s, span %d, bci %d, method %s", getPosStr(), getSpan(), frame.getBCI(), frame.getMethod().format("%h.%n(%p)"));
         }
 
+        public String getLocalsStr() {
```

Comment:
@adinn this is a hopefully more correct example code how to get locals info for the `CompilationResultFrameTree`